src/mcp/prompts/crashlytics/connect.ts

```diff
@@ -24,19 +24,25 @@ Active user: ${accountEmail || "<NONE>"}
 ## Required first steps! Absolutely required! Incredibly important!
 
   1. **Make sure the user is logged in. No Crashlytics tools will work if the user is not logged in.**
-    a. Use the \`firebase_get_environment\` tool to verify that the user is logged in,
-       and find the active Firebase project.
+    a. Use the \`firebase_get_environment\` tool to verify that the user is logged in.
     b. If the Firebase 'Active user' is set to <NONE>, instruct the user to run \`firebase login\` 
-       before continuing. 
+       before continuing. Ignore other fields that are set to <NONE>. We are just making sure the
+       user is logged in. 
 
   2. **Get the app_id for the Firebase application.** 
```

Comment:
Which file are we referring to here? Should we say the list of possible options for the Google-Services file?

Just to be clear, we do specify the name below. But we are referencing to the concept of a file before those steps - so the curious suggestion.